.github/CODEOWNERS

```diff
@@ -1,14 +1,16 @@
+# GENERATED by @kbn/dev-utils (Do not edit manually)
```

Comment:
Can we additionally put a note that this list is configured in `packages/kbn-dev-utils/src/code_ownership/ownership_config.ts` to prevent users wanting to modify it having to hunt around?